style.css

```diff
@@ -0,0 +1,48 @@
+@font-face {
```

Comment:
All your customized CSS style sheets should be placed inside their own folder called ``styles.``